syntax = "proto2";
package protos;

enum LossType {
    CLASSIFICATION = 0;
    L2_LOSS = 1;
    SIGMOID_L2_LOSS = 2;
    // crossentropy loss/log loss
    CROSS_ENTROPY_LOSS = 3;
    SOFTMAX_CROSS_ENTROPY = 4;
}
